DER/FxDerivatives/FxContracts.rdf

```diff
@@ -75,132 +75,134 @@
+				<owl:someValuesFrom rdf:resource="&fibo-fnd-acc-cur;ExchangeRate"/>
+			</owl:Restriction>
+		</rdfs:subClassOf>
+		<rdfs:label xml:lang="en">currency forward</rdfs:label>
 		<skos:definition xml:lang="en">agreement to deliver and settle a given amount of currency in one currency, in exchange for a given amount in another currency, at an agreed date in the future and at an agreed rate of exchange</skos:definition>
+		<fibo-fnd-utl-av:synonym xml:lang="en">foreign exchange forward</fibo-fnd-utl-av:synonym>
 	</owl:Class>
 	
-	<owl:Class rdf:about="&fibo-der-fx-ctr;ForeignExchangeForwardOutright">
-		<rdfs:subClassOf rdf:resource="&fibo-der-fx-ctr;ForeignExchangeForward"/>
+	<owl:Class rdf:about="&fibo-der-fx-ctr;CurrencyForwardOutright">
+		<rdfs:subClassOf rdf:resource="&fibo-der-fx-ctr;CurrencyForward"/>
 		<rdfs:subClassOf>
 			<owl:Restriction>
 				<owl:onProperty rdf:resource="&lcc-cr;hasPart"/>
-				<owl:someValuesFrom rdf:resource="&fibo-der-fx-ctr;ForeignExchangeSpotContract"/>
+				<owl:someValuesFrom rdf:resource="&fibo-der-fx-ctr;CurrencySpotContract"/>
 			</owl:Restriction>
 		</rdfs:subClassOf>
 		<rdfs:subClassOf>
 			<owl:Restriction>
 				<owl:onProperty rdf:resource="&lcc-cr;hasPart"/>
-				<owl:someValuesFrom rdf:resource="&fibo-der-fx-ctr;ForeignExchangeSwap"/>
+				<owl:someValuesFrom rdf:resource="&fibo-der-fx-ctr;CurrencySwap"/>
 			</owl:Restriction>
 		</rdfs:subClassOf>
-		<rdfs:label xml:lang="en">foreign exchange forward outright</rdfs:label>
-		<skos:definition xml:lang="en">forward contract in the forex market that locks in the price at which an entity can buy or sell a currency on a future date. Also known as &quot;outright forward currency transaction&quot;, &quot;forward outright&quot; or &quot;FX forward&quot;.</skos:definition>
-		<fibo-fnd-utl-av:explanatoryNote xml:lang="en">In currency forward contracts, the contract holders are obligated to buy or sell the currency at a specified price, at a specified quantity and on a specified future date. These contracts cannot be transferred. Jan 10 Review Notes Outright Forward is the term for the professional markets. Spot + Swap where Swap is 2 simultaneous transactions.</fibo-fnd-utl-av:explanatoryNote>
+		<rdfs:label xml:lang="en">currency forward outright</rdfs:label>
+		<skos:definition xml:lang="en">forward contract in the forex market that locks in the price at which an entity can buy or sell a currency on a future date</skos:definition>
+		<fibo-fnd-utl-av:explanatoryNote xml:lang="en">The holders of a currency forward are obligated to buy or sell the currency at a specified price, at a specified quantity and on a specified future date. These contracts cannot be transferred. Jan 10 Review Notes Outright Forward is the term for the professional markets. Spot + Swap where Swap is 2 simultaneous transactions.</fibo-fnd-utl-av:explanatoryNote>
+		<fibo-fnd-utl-av:synonym xml:lang="en">FX forward</fibo-fnd-utl-av:synonym>
```

Comment:
No this is a synonym for the superclass

Reply:
fixed, although investopedia said that it was a synonym for the outright